### DISCUSSION TITLE
Trakt.tv

### DISCUSSION THREAD

**Discussion by KibosJ** [elrumo/macOS_Big_Sur_icons_replacements#1129] [Icon request] [answered]
Would someone be able to do a Big Sur style icon for trakt.tv please?

[Trakt.tv](https://trakt.tv)

**Comment by colineberly** [comment, marked as answer]
Here's a few I did up this morning.

![Trakt](https://user-images.githubusercontent.com/61343/113511810-ff512e80-951e-11eb-8c36-358ee9ab3469.png)
![Trakt-Alt](https://user-images.githubusercontent.com/61343/113511813-00825b80-951f-11eb-9c01-63503c9bf872.png)
![Trakt-Alt-v2](https://user-images.githubusercontent.com/61343/113511812-ffe9c500-951e-11eb-85da-966f620931fb.png)

**Comment by cake03** [comment]
@KibosJ, Please close this [Discussion](.)
